app/views/mentors/show.html.erb

```diff
@@ -10,7 +10,9 @@
 
 <h3>Missions proposed :</h3>
 <ul>
-  <li>Mission 1 : xxx</li>
+  <% @missions.each_with_index do |mission, index| %>
+    <li>Mission <%= index + 1 %> : <em><%= mission.name %></em></li>
```

Comment:
Perfect Joe, i think we could add more infos about the mission (company, fee, etc..)